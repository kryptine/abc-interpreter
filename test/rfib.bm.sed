s/26\.0/41.0/
