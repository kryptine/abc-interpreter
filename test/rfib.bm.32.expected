535828544
halt at 6
3 524285 524288
